src/main/java/com/thebois/models/world/IWorld.java

```diff
@@ -0,0 +1,38 @@
+package com.thebois.models.world;
+
+import com.thebois.models.Position;
+
+/**
+ * Represents a world containing tiles.
+ */
+public interface IWorld {
```

Comment:
I hope we can look into merging this functionality with IFinder later. I think it would fit perfectly, some functionality is already implemented (in my branch)

Reply:
Yup, that's probably a good idea!

(this is a comment, not a request for change :P )

Reply:
Also, you can start your review, leave all your comments/requests for changes, and then submit them all at once, instead of reviewing multiple times!

Reply:
But I love making multiple reviews! Also it is "important Feedback you must address before merging" 😉